bindata/v4.1.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -106,6 +106,11 @@ servingInfo:
   requestTimeoutSeconds: 3600
 serviceAccountPublicKeyFiles:
   - /etc/kubernetes/static-pod-resources/configmaps/sa-token-signing-certs
+  - /etc/kubernetes/static-pod-certs/configmaps/bound-sa-token-signing-certs
+serviceAccountIssuer: auth.openshift.io
+apiAudiences:
+  - auth.openshift.io
+serviceAccountSigningKeyFile: /etc/kubernetes/static-pod-certs/secrets/service-account-signing-key/service-account.key
```

Comment:
isn't there are need to override this for bootstrapping? Or do we get this key from the render step that early?

Reply:
The intent is not to enable bound tokens in the bootstrapping phase, since as per a [discussion](https://github.com/openshift/enhancements/pull/150/files#r357266863) on the enhancement there does not appear to be a need for that. Will it be necessary to set these values in code that can detect that bootstrapping is complete rather than here?

Reply:
I think we're ok not having bound tokens available from the bootstrap kubeapiserver.  we asked in the enhancement.

Reply:
My question was more whether this path is a valid one during bootstrap or whether the process dies if not.

Reply:
Ah, ok.  Will test tomorrow. Maybe the operator will have to detect that it is past the bootstrap phase and compose the config accordingly in code?

Reply:
There are two override yaml files, one for the bootstrap phase, one for after. Just set sensible values for each.